File: src/main/java/com/hedvig/backoffice/services/members/dto/InsuranceCancellationDTO.java
package com.hedvig.backoffice.services.members.dto;

import java.time.LocalDate;
import java.util.UUID;
import lombok.Value;

@Value
public class InsuranceCancellationDTO {
  Long memberId;
  UUID insuranceId;
  LocalDate cancellationDate;
}

File: src/main/java/com/hedvig/backoffice/services/members/dto/InsuranceCancellationDTO.kt
package com.hedvig.backoffice.services.members.dto

import java.time.LocalDate
import java.util.*

data class InsuranceCancellationDTO (
  val memberId: Long? = null,
  val insuranceId: UUID? = null,
  val cancellationDate: LocalDate? = null
)
